docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -121,7 +120,7 @@ Complete the following steps to upgrade the server:
 1. Run the fhir-installer on a separate server.  
 2. Configure the new server as appropriate (`fhir-server.xml` and anything under the `fhir-server/config` and `fhir-server/userlib` directories).  
```

Comment:
fhir-server.xml? or server.xml?

Reply:
good catch.  This wasn't even a line I changed, but I'm pretty sure that should be `server.xml`.
```suggestion
2. Configure the new server as appropriate (`server.xml` and anything under the `fhir-server/config` and `fhir-server/userlib` directories).  
```

Reply:
should be addressed now